src/main/scala/cognite/spark/v1/AlphaDataModelInstancesHelper.scala

```diff
@@ -347,5 +351,31 @@ object AlphaDataModelInstancesHelper {
       case _ =>
         throw SparkSchemaHelperRuntime.badRowError(row, keyString, "String", "")
     }
+
+  def getDirectRelationIdentifierProperty(
+      externalId: String,
+      row: Row,
+      keyString: String,
+      index: Int): DirectRelationIdentifier =
+    row.get(index) match {
+      case identifier: String =>
+        // Colon character cannot be used for space but it can be used for the externalId, so we should consider only
+        // the first ":" to split between spaceExternalId and externalId
+        identifier.split(":", 2) match {
+          case Array(spaceExternalId, externalId) =>
+            DirectRelationIdentifier(Some(spaceExternalId), externalId)
+          case _ =>
+            val errorHint = if (identifier.isBlank) {
+              ""
+            } else {
+              s", but was: $identifier"
+            }
+            throw new CdfSparkException(
+              s"Invalid data model instance row with external id '$externalId'. The values " +
```

Comment:
Should avoid using `+` for string concatenation, see https://nrinaudo.github.io/scala-best-practices/tricky_behaviours/string_concatenation.html